release.sh

```diff
@@ -1,9 +1,23 @@
+echo "Starting Release for Environment '$ENV'"
+
+# ----------- STAGING & REVIEW APPS --------------
+if [ "$ENV" = 'dev' ]; then
+  echo "=========== Development Release =============="
+  echo "----------- Set up DB Schema -----------------"
+  npm run db:setup:heroku
+  npm run db:reset-hard
+  echo "----------- Seed DB with sample users --------"
+  npm run db:seed
+  echo "=========== DONE ============================="
+fi
+
+# ----------- PRODUCTION -------------------------
 if [ "$ENV" = 'production' ]; then
-    echo 'Production Deployment, running migrations'
-    npx typeorm migration:run
-else 
-    echo 'Non-Productive Deployment, setting up DB with seeded data' 
```

Comment:
Why not if else anymore?

Reply:
I wanted to explicitly check the value of ENV, and then the branches are mutually exclusive anyways